src/widget.lisp

```diff
@@ -78,9 +78,28 @@ inherits from REBLOCKS/WIDGET:WIDGET if no DIRECT-SUPERCLASSES are provided."
 
 
 (defgeneric get-html-tag (widget)
-  (:documentation "This method should return a keyword, like :div or :article.
-                   By default, it returns :div.
-                   If we are inside a table, it returns :tr"))
+  (:documentation "This method determines the enclosing tag of the widget.
+The return value should either be a keyword like :div, which will be the enclosing tag,
+or a list of the form (:tag . attributes), where :tag is the enclosing tag (like :div)
+and attributes is a property list, where the keys are keywords, corresponding to the attribute name
+and the values are the values of the attribute.
+
+For example:
+
+- :div  -- generates a <div ...> WIDGET CONTENT </div>
+- (:div :display \"flex\") -- generates (<div ... :display \"flex\">WIDGET CONTENT</div>
```

Comment:
To fix documentation building, make these words lowercase. Otherwise, documentation builder tries to find these symbols in the current package for cross-referencing.